### PR TITLE
Version updates for mods

### PR DESCRIPTION
**Pull request by Enscivwy**
Updated the following mods:
Fabric API
Sodium
Mod Menu
Indium
KennyTVsEpicForceCloseLoadingScreenMod
Iris
Physics Mod
Shoulder Surfing Reloaded
BetterF3
Server Packer Unlocker
Sodium Extra
NBTToolTips
ArchitecturyAPI
FabricKotlinLanguage
Yet Another Config Lib
Continuity

### CODE REVIEW

File: versions.yml [hunk 1]
```diff
@@ -280,7 +280,7 @@ modGroups:
       license: MIT
       homepage: https://www.curseforge.com/minecraft/mc-mods/shoulder-surfing-reloaded
       desc: Revamped over-the-shoulder F5 third-person camera view
-      url: https://mediafilez.forgecdn.net/files/4443/980/ShoulderSurfing-Fabric-1.19.4-2.3.0.jar
+      url: https://mediafilez.forgecdn.net/files/4510/888/ShoulderSurfing-Forge-1.19.4-2.3.2.jar
```
Comment (kristofbolyai) [inline versions.yml]: You need the fabric version.

File: versions.yml [hunk 2]
```diff
@@ -317,7 +317,7 @@ modGroups:
       homepage: https://modrinth.com/mod/betterf3
       modrinthId: 8shC1gFX
       desc: Makes the F3-menu customizable and cleaner
-      url: https://cdn.modrinth.com/data/8shC1gFX/versions/jFVjbyGK/BetterF3-6.0.0-Fabric-1.19.4-pre3.jar
+      url: https://cdn.modrinth.com/data/8shC1gFX/versions/cEflCp0r/BetterF3-6.0.1-Forge-1.19.4.jar
```
Comment (kristofbolyai) [inline versions.yml]: Fabric version here too.

File: versions.yml [hunk 3]
```diff
@@ -442,15 +442,15 @@ modGroups:
       homepage: https://modrinth.com/mod/architectury-api
       modrinthId: lhGA9TYQ
       desc: Dependency for RoughlyEnoughItems
-      url: https://cdn.modrinth.com/data/lhGA9TYQ/versions/tJGqxo3S/architectury-8.1.79-fabric.jar
+      url: https://cdn.modrinth.com/data/lhGA9TYQ/versions/oGBEjb15/architectury-8.1.80-forge.jar
```
Comment (kristofbolyai) [inline versions.yml]: Fabric version

Comment (Enscivwy) [reply versions.yml]: Sorry, version changed

**Comment by Enscivwy** [comment]
Physics mod version was reverted due to crashing

**Review by DevScyu — APPROVED** [review]

**Pull request merged**